gen3/bin/kube-setup-fluentd.sh

```diff
@@ -16,15 +16,30 @@ ctx="$(g3kubectl config current-context)"
 ctxNamespace="$(g3kubectl config view -ojson | jq -r ".contexts | map(select(.name==\"$ctx\")) | .[0] | .context.namespace")"
 # only do this if we are running in the default namespace
 if [[ "$ctxNamespace" == "default" || "$ctxNamespace" == "null" ]]; then
-  if (! g3kubectl --namespace=kube-system get daemonset fluentd > /dev/null 2>&1) || [[ "$1" == "--force" ]]; then
```

Comment:
should we undeploy the fluentd daemonset in `kube-system` if it exists?

Reply:
The instruction is already there, perhaps you missed it https://github.com/uc-cdis/cloud-automation/pull/1238/files#diff-e9e44415744c2b19024b113d2d176dbaR37